File: back/DirectorgBaze/Models/Director.cs
using System;
using System.Collections.Generic;
using System.Linq;
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System.Threading.Tasks;

namespace DirectorgBaze.Models
{
    public class Director
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; }

        public string Name { get; set; }

        public string Surname { get; set; }

        public string Email { get; set; }

        public string Sertificate { get; set; }
    }
}

File: front/src/app/models/Director.ts

export interface IDirector {
    id: string;
    name:string;
    surname:string;
    email: string;
    sertificate: string;
}

export class Director implements IDirector{
    id: string;
    name:string;
    surname:string;
    email: string;
    sertificate: string;

    constructor(name, surname, email, sertificate) {
        this.name=name;
        this.surname=surname;
        this.email=email;
        this.sertificate=sertificate;
    }
  
}